vendor/github.com/mattn/go-shellwords/.travis.yml

```diff
@@ -2,7 +2,6 @@ language: go
 go:
   - tip
 before_install:
-  - go get github.com/axw/gocov/gocov
```

Comment:
> Perhaps you're using an older version

Old version of trash? Just removed and updated via go get. And don't you mention:

> When I run trash this line gets added back

i.e. don't you mean that you have same situation locally that it takes master instead v1.0.0?

Also I did neither touch trash.conf nor run `trash` at all